<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderAccountDialog</class>
 <widget class="QDialog" name="ReaderAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="id_edt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="id_lbl">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编    号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_lbl">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓    名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="phone_lbl">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手机号码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="type_lbl">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>读者类型：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="stat_lbl">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>读者状态：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bowwow_num_lbl">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>已借数目：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_edt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="borrow_num_edt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="type_cbox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>学生</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>教师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外来</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="stat_edt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>正常</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>限制</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="new_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新 建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保 存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删 除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
